opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -124,7 +127,7 @@ def __init__(
 
         self._description = description
         self._attribute_keys = attribute_keys
-        self._aggregation = aggregation
+        self._aggregation = aggregation or DefaultAggregation()
```

Comment:
nit we can use a single cached instance of `DefaultAggregation`

Reply:
REady